### DISCUSSION TITLE
how watching release notes latest version?

### DISCUSSION THREAD

**Discussion by luckypong** [unoplatform/uno#15722] [Q&A]
Nuget versions are constantly being updated.  Are there any release notes for the latest Nuget version?

**Comment by limeniye** [comment]
Hello, @luckypong! Let's check the releases list of Uno library there: https://github.com/unoplatform/uno/releases

**Reply by luckypong** [reply]
Hello @limeniye 
I would like to see the release notes for nuget latest version 5.1.75

**Reply by limeniye** [reply]
I don't need it, because sometimes I subscribe to the interesting issues for me, and I know that the last one was that: https://github.com/unoplatform/uno/issues/13470

**Reply by jeromelaban** [reply]
@luckypong We do not publish the release notes for intermediate versions as github releases, but you can find the changes by looking at the branch here: https://github.com/unoplatform/uno/commits/release/stable/5.1
